platform/ios/docs/guides/For Style Authors.md

```diff
@@ -253,9 +271,9 @@ In style JSON | In Objective-C | In Swift
 
 Each property representing a layout or paint attribute is set to an
 `MGLStyleValue` object, which is either an `MGLConstantStyleValue` object (for
-constant values) or an `MGLStyleFunction` object (for zoom level functions). The
-style value object is a container for the raw value or function parameters that
-you want the attribute to be set to.
+constant values) or an `MGLStyleFunction` subclass. The style value object is a
+container for the raw value or function parameters that you want the attribute
+to be set to.
```

Comment:
Per https://github.com/mapbox/mapbox-gl-native/pull/8025#discussion_r101864980, let’s take this opportunity to point out that not every interpolation mode is available for every attribute. The developer should consult the documentation for an individual layout or paint attribute to see which function types and interpolation modes it supports.